cli/commands/preview/update.ts

```diff
@@ -33,12 +43,6 @@ function getSnapshotToUpdate(
 		);
 	}
 
-	const now = new Date();
-	const endDate = addDays( snapshotToUpdate.date, DEMO_SITE_EXPIRATION_DAYS );
-	if ( endDate < now ) {
-		throw new LoggerError( __( 'Cannot update an expired preview site.' ) );
-	}
-
```

Comment:
This logic is actually repeated in `runCommand`. I know I previously recommended extracting it into this standalone function, but coming back to it now, I feel it's better to keep it in `runCommand` after all.